inst/extdata/abm.Rmd

```diff
@@ -208,10 +208,15 @@ if(wospubs < 10)
 wospubstext <- paste0(wospubstext, "</p>")
 cat(wospubstext)
 
-coveragetext <- paste0("<p>Web of Science coverage is ", 100*round(totcov, 3), "%. ")
-if(totcov >= 0.8) coveragetext <- paste0(coveragetext, "This is OK!</p>")
-if(totcov < 0.8 && totcov >= 0.6) coveragetext <- paste0(coveragetext, "<b>This is bad, proceed with caution!</b></p>")
-if(totcov < 0.6) coveragetext <- paste0(coveragetext, "<b>This is very bad, indicators can not be trusted!</b></p>")
+coveragejudge <- ifelse(totcov >= 0.8, "good",
+                        ifelse(totcov >= 0.65, "moderate",
+                               ifelse(totcov >= 0.5, "poor", "bad")))
```

Comment:
compact way that works, an alternative is to use https://adv-r.hadley.nz/control-flow.html#switch here

Reply:
Fetching from function in latest version, so not a current concern. Does switch work with intervals?

Reply:
This SO post seems to say ifelse may be preferred if not `cut` is even better? 

https://stackoverflow.com/questions/30293465/in-r-how-to-interpret-switch-statement-with-greater-than-less-than

```r
totcov <- 0.3
as.character(cut(totcov, right = F,
 c(0, 0.5, 0.65, 0.8, 1), 
 c("bad", "poor", "moderate", "good")))
```

Or for reuse:

```r
cvrg <- runif(10)  # one or several

judge <- function(x) {
  if (is.na(x) || is.null(x)) return (NA)
  as.character(cut(x, right = FALSE,
    breaks = c(0, 0.5, 0.65, 0.8, 1),
    labels = c("bad", "poor", "moderate", "good")
  ))
}

judge(cvrg)
judge(cvrg[1])
judge(NA)
judge(NULL)
```

Reply:
it already works well, right?